src/app/shared/enums/block-type.enum.ts

```diff
@@ -10,4 +10,8 @@ export enum BlockType {
   MultiSelectInput = 'multi-select-input',
   ShortTextInput = 'short-text-input',
   SectionHeading = 'section-heading',
+
+  // [NS] Currently not used
+  ContributorsInput = 'contributors-input',
+  SelectOtherOption = 'select-other-option',
```

Comment:
We can remove this, as these haven't been used and I don't think there are plans to use this in the future.